joshuaCarpentier/assignments/treesBFS/lc104/lc104.js

```diff
@@ -10,10 +10,58 @@ class TreeNode {
   }
 }
 
+/**
+ * 
+ * Input: tree
+ * Output: the maximum depth level
+ * 
+ * Variables: root, root. left, root.right, index, array to store the root 
 
+ * 
+ * Brute force Approach
+ * --------------- 
+ * Pre-Condition: check if empty 
+ * 
+ * Conditions: 
+ * 
+ * Termination-Conditions: if there is only a root node return level
+
+ * 
+ * Post-Conditions: 
+ * 
+ * there is only really a pre condition since were only going to lowest level
+ * 
+ * 
```

Comment:
For an approach, it would be better if you detailed what you code was going to do and how in sentences. By looking at this approach, I am not sure what your solution is going to do or how it will solve the prompt.